code/modules/clothing/masks/hailer.dm

```diff
@@ -2,7 +2,7 @@
 // **** Security gas mask ****
 
 /obj/item/clothing/mask/gas/sechailer
-	name = "security gas mask"
+	name = "Security Compli-o-nator"
```

Comment:
Aw

Reply:
It's a result of me changing the output to use [src] so I can copy the proc, I can change name back if you want but the text phrase won't look as cool in chat then

Reply:
Your way makes more sense, actually